config.go

```diff
@@ -33,6 +33,17 @@ type Process struct {
 	Cwd string `json:"cwd"`
 }
 
+// User specifies linux specific user and group information for the container's
+// main process.
+type User struct {
+	// UID is the user ID the Process is executed as. (this field is platform dependent)
```

Comment:
Should we use a json tag to indicate platform dependency? It will help with tooling as well.